src/hist/plot.py

```diff
@@ -72,26 +73,29 @@ def _expr_to_lambda(expr: str) -> Callable:
         varnames.append(tokval)
     varnames = list(OrderedDict.fromkeys([name for name in varnames if name != "x"]))
     lambdastr = f"lambda x,{','.join(varnames)}: {expr}"
-    return eval(lambdastr)
+    return eval(lambdastr)  # type: ignore
 
 
 def _curve_fit_wrapper(
-    func: Callable,
-    xdata: ArrayLike,
-    ydata: ArrayLike,
-    yerr: ArrayLike,
+    func: Callable[..., Any],
+    xdata: np.ndarray,
+    ydata: np.ndarray,
+    yerr: np.ndarray,
     likelihood: bool = False,
-) -> Tuple[ArrayLike, ArrayLike]:
+) -> Tuple[Tuple[float, ...], ArrayLike]:
```

Comment:
Makes the typing easier, because NumPy's typing is a bit spotty. It doesn't know that `*array` is valid, etc. This is small so was a simple fix.